templates/types/streaming/express/package.json

```diff
@@ -20,7 +20,8 @@
     "ajv": "^8.12.0",
     "@e2b/code-interpreter": "^0.0.5",
     "got": "10.7.0",
-    "@apidevtools/swagger-parser": "^10.1.0"
+    "@apidevtools/swagger-parser": "^10.1.0",
+    "form-data": "^4.0.0"
```

Comment:
@thucpn have you tried the node internal one: https://nodejs.org/api/globals.html#class-formdata? - we should not use a three year old package